modules/statistics/templates/form_stats_behavioural.tpl

```diff
@@ -55,7 +55,7 @@
 
 
 
-    <b><a href='{$baseurl}/statistics/statistics_site/?CenterID={$CurrentSite.ID}&ProjectID={$CurrentProject.ID}' target="_blank">Click here for breakdown per participant {if $CurrentSite} for {$CurrentSite.Name} {/if} {if $CurrentProject} for {$CurrentProject.Name} {/if}</a></b>
+    <b><a href='{$baseurl}/statistics/statistics_site/?CenterID={$CurrentSite.ID}&ProjectID={$CurrentProject.ID}' target="_blank">Per Instrument Stats - All Sites</a></b>
```

Comment:
```suggestion
    <b><a href='{$baseurl}/statistics/statistics_site/?CenterID={$CurrentSite.ID}&ProjectID={$CurrentProject.ID}' target="_blank">Per Instrument Data Entry Stats - All Sites</a></b>
```

Reply:
Agreeing with @racostas ' review : here's the suggestion/change requested.

Reply:
@christinerogers I have resolved the mentioned issues!